packages/p2p/src/components/my-profile/payment-methods/add-payment-method/select-payment-method.jsx

```diff
@@ -46,7 +46,7 @@ const SelectPaymentMethod = () => {
                                 <Text key={0} color='less-prominent' size='xxs' />,
                                 <Text
                                     key={1}
-                                    color='loss-danger'
+                                    className='link'
```

Comment:
You can add `className='link'` but please don't remove `color='loss-danger'` since as per design, the text color is red as well.

Reply:
If the text color is `#ff444f` (--brand-coral-red), then `color='loss-danger'` is not required since `link` already has it. If it is `#ec3f3f` (--text-loss-danger), then `color='loss-danger'` shouldn't be removed.

Reply:
In the initial change , I had removed loss-danger thinking about uniformity because in our application I could see link being used at many places where such links are used.  In the latest commit I have kept color as loss danger as it was before 😄  along with the other styles.